### PR TITLE
xcursor themes: added the comix, flatbed, gruppled, polar and premium xcursor themes

### PR DESCRIPTION
**Pull request by msteen**
###### Motivation for this change
I wanted a different xcursor theme, but could not find any that I liked available.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/icons/polar-xcursor-theme/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "${package-name}-${version}";
+  package-name = "polar-xcursor-theme";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://dl.opendesktop.org/api/files/download/id/1460735356/27913-PolarCursorThemes.tar.bz2";
```
Comment (Mic92) [inline pkgs/data/icons/polar-xcursor-theme/default.nix]: The hash of this theme changed to `167m2q6fb1warzwr836jbhjdiddhzhnzskhd4kmhgxc9ii97rmq3`.
Is there a more stable url?

Comment (Mic92) [inline pkgs/data/icons/polar-xcursor-theme/default.nix]: Otherwise can you ask the author to upload one file per release?

**Comment by msteen** [comment]
In the meantime other xcursor themes I had packaged have had unstable source URLs as well, even though some of them should not have been updated in years. So I will close this pull request, because I am not planning on supplying a mirror for them myself.

**Comment by nim65s** [comment]
For reference, https://github.com/NixOS/nixpkgs/pull/242667 is adding gruppled, re-uploaded on github

**Pull request closed without merging**